src/jit/codegenlinear.h

```diff
@@ -134,6 +134,8 @@ void genFMAIntrinsic(GenTreeHWIntrinsic* node);
 void genLZCNTIntrinsic(GenTreeHWIntrinsic* node);
 void genPCLMULQDQIntrinsic(GenTreeHWIntrinsic* node);
 void genPOPCNTIntrinsic(GenTreeHWIntrinsic* node);
+void genJumpTableFallback(
```

Comment:
I think @sdmaclea had a good approach to this in one of his PRs.

Specifically, the codegen is essentially the same except for the actual emitted instruction and could be handled via a lambda.

Reply:
https://github.com/dotnet/coreclr/pull/16085/files#diff-ddd0dded7caa705d00fda562d726c37aR5180
and
https://github.com/dotnet/coreclr/pull/16085/files#diff-ddd0dded7caa705d00fda562d726c37aR5266